src/Server.ts

```diff
@@ -21,14 +24,23 @@ class Server{
   }
```

Comment:
Use "this.config.port" instead of "this.port " or you can destructure "this.config" to get port.
- Learn about destructuring in node js